setup.py

```diff
@@ -24,12 +24,6 @@
     url='https://github.com/nasa-cumulus/cumulus-process-py',
     license='Apache 2.0',
     classifiers=[
-        'Topic :: Software Development :: Libraries',
```

Comment:
is there a reason to remove these, aside from the python 3.8?

Reply:
because twine, the package we use to upload to pipit, trips on multiple classifiers.
I can't find any record of why, or what is a version that doesn't have this bug, I can't find anyone complaining about it online, but I dug into the source code and found that it parses these params as tuples, with key as first value, and value as second value, and if we have multiple ("classifiers": "thing") tuples the requests package can't handle multiple entries without he same key. but pypi won't let me *fix* a package if it's uploaded mangled (I have to release it as 1.2.3.patch or something) so I don't want to mess about without being able to proper test it, and this seems to be not an error in twine but a misconfiguration between specific twine and requests versions, but after a bit of mucking about I couldn't find the specific relationship.

all of this is to say, if you feel like this is important I can investigate further how to keep these classifiers

Reply:
I have no idea if they're important or not, but was wondering why they were being removed. I think for something like this, given the day, maybe bring it up at Arch meeting and see what folks think. I imagine our options are 1) these are not important and we can do without, 2) we need to keep these so we can maybe remove them for now and have a ticket to figure out how to get the different packages to play nicely together later, or 3) we need to figure out how to keep them for this release. I would think 3 is less likely but I really don't know.